eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Internal.hs

```diff
@@ -433,12 +435,19 @@ votingCommitteeThresholdInternal pp committee = \case
   where
     threshold =
       case committeeThreshold <$> committee of
-        -- if the committee size is smaller than the mnimimum given in pparams,
-        -- we treat it as if we had no committe
-        SJust t | committeeSize >= minSize -> VotingThreshold t
+        -- if the committee size is smaller than the minimum given in PParams,
+        -- we treat it as if we had no committee
+        SJust t | activeCommitteeSize >= minSize -> VotingThreshold t
         _ -> NoVotingThreshold
     minSize = pp ^. ppCommitteeMinSizeL
-    committeeSize = fromIntegral $ Map.size $ foldMap' committeeMembers committee
+    isActive coldKey _validUntil =
+      case Map.lookup coldKey hotKeys of
+        Just (CommitteeMemberResigned _) -> False
+        Just _ -> True -- TODO do we need to check if member's term is expired here?
```

Comment:
I tried to find the answer to this question in the spec, but not sure.
I see this 
[actualCCVotes](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12549) ([just](https://intersectmbo.github.io/formal-ledger-specifications/html/Agda.Builtin.Maybe.html#173) ([cc](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12653) [,](https://intersectmbo.github.io/formal-ledger-specifications/html/Agda.Builtin.Sigma.html#235) [q](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12658)))  =  [if](https://intersectmbo.github.io/formal-ledger-specifications/html/Interface.ToBool.html#282) [ccMinSize](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.PParams.html#3075) [≤](https://intersectmbo.github.io/formal-ledger-specifications/html/Interface.HasOrder.html#301) [lengthˢ](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Set.Theory.html#724) ([activeCC](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12051) ([just](https://intersectmbo.github.io/formal-ledger-specifications/html/Agda.Builtin.Maybe.html#173) ([cc](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12653) [,](https://intersectmbo.github.io/formal-ledger-specifications/html/Agda.Builtin.Sigma.html#235) [q](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12658))))

where 
[activeCC](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12051) ([just](https://intersectmbo.github.io/formal-ledger-specifications/html/Agda.Builtin.Maybe.html#173) ([cc](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12102) [,](https://intersectmbo.github.io/formal-ledger-specifications/html/Agda.Builtin.Sigma.html#235) _))  = [dom](https://intersectmbo.github.io/formal-ledger-specifications/html/Interface.IsSet.html#794) ([filter](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#321) (λ (_ [,](https://intersectmbo.github.io/formal-ledger-specifications/html/Agda.Builtin.Sigma.html#235) [x](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12134)) → [Is-just](https://intersectmbo.github.io/formal-ledger-specifications/html/Data.Maybe.html#810) [x](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12134)) ([ccHotKeys](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#7239) [∣](https://intersectmbo.github.io/formal-ledger-specifications/html/Axiom.Set.Map.html#9375) [dom](https://intersectmbo.github.io/formal-ledger-specifications/html/Interface.IsSet.html#794) [cc](https://intersectmbo.github.io/formal-ledger-specifications/html/Ledger.Ratify.html#12102)))

but nothing about expiration..

Reply:
Yeah, I noticed that also. IIRC we remove expired CC members at the epoch boundary so we don't have to worry about them here, but I haven't confirmed that.

Reply:
No, we do not remove expired CC members on the epoch boundary. Any member can only be removed with an explicit governance action.

On the epoch boundary we remove hot credential authorizations that do not belong to any existing cold credential, but that is modification of the committee state. The actual committee members can only be adjusted with `UpdateCommittee` proposal.